<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CGAL Terrain</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resources/CGAL.qrc" >
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="Viewer" native="1" name="viewer" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionLoad_New_File" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_New_File" >
   <property name="text" >
    <string>Load New File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/CGAL.qrc" />
  <include location="Terrain.qrc" />
 </resources>
 <connections/>
</ui>
